packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -701,6 +701,10 @@ class RenderParagraph extends RenderBox with ContainerRenderObjectMixin<RenderBo
       .maxIntrinsicWidth;
   }
 
+  /// An estimate of the height of a line in the text. See [TextPainter.preferredLineHeight].
+  /// This does not require the layout to be updated.
+  double get preferredLineHeight => _textPainter.preferredLineHeight;
```

Comment:
add `@visibleForTest`?

Reply:
`RenderEditable` has this API public, I don't have a strong preference either way, what do you think?

Reply:
I will add it just to be conservative, we can always move to public in the future, but not backward

Reply:
Sounds good to me!